cluster/examples/kubernetes/ceph/cluster-external.yaml

```
@@ -20,3 +20,8 @@ spec:
     enable: true
   crashCollector:
     disable: true
+  # optionally, the ceph-mgr IP address can be pass to gather metric from the prometheus exporter
```

```suggestion
  # optionally, the ceph-mgr IP address can be passed to gather metric from the prometheus exporter
```

and shouldn't this be at Line 26?